samples/excel/Basics/range.yaml

```diff
@@ -9,14 +9,40 @@ script:
     $('#setup').click(setup);
 
     $('#set-formulas').click(setFormulas);
+    $('#set-formulas-R1C1').click(setFormulasR1C1);
+    $('#get-values').click(getValues);
     $('#get-texts').click(getTexts);
+    $('#get-formulas').click(getFormulas);
 
     function setup() {
         Excel.run(async (context) => {
-            let sheet = context.workbook.worksheets.getActiveWorksheet();
+            let oldSheet = context.workbook.worksheets.getActiveWorksheet();
+            oldSheet.load(["name", "position"]);
```

Comment:
I agree. I tried exactly that and I ran into the issue with the context object missing a workbook property. One that is resolved I can move the code to reset the workhsheet into a function.